packages/webapp/public/locales/es/translation.json

```diff
@@ -539,34 +539,34 @@
         "TEMPERATURE": "temperatura"
       },
       "VALIDATION": {
-        "FILE_ROW_LIMIT_EXCEEDED": "MISSING - Es - File row limit exceeded. The max number of sensors that can be uploaded from a file is 100.",
-        "MISSING_COLUMNS": "MISSING - Es - Columns are required/missing.",
-        "EXTERNAL_ID": "MISSING - Es - Invalid external id, must be between 1 and 20 characters.",
-        "SENSOR_NAME": "MISSING - Es - Invalid sensor name, must be between 1 and 100 characters.",
-        "SENSOR_LATITUDE": "MISSING - Es - Invalid latitude value, must be between -90 and 90. and fewer than 10 decimals.",
-        "SENSOR_LONGITUDE": "MISSING - Es - Invalid longitude value, must be between -180 and 180. and fewer than 10 decimals.",
-        "SENSOR_READING_TYPES": "MISSING - Es - Invalid reading type detected, valid values include: soil_water_content, soil_water_potential, temperature.",
-        "SENSOR_DEPTH": "MISSING - Es - Invalid depth value, must be between 0 and 1000.",
-        "SENSOR_BRAND": "MISSING - Es - Invalid brand name, must be between 1 and 100 characters.",
-        "SENSOR_MODEL": "MISSING - Es - Invalid model name, must be between 1 and 100 characters.",
-        "SENSOR_HARDWARE_VERSION": "MISSING - Es - Invalid hardware version name, must be between 1 and 100 characters."
+        "FILE_ROW_LIMIT_EXCEEDED": "Límite de fila de archivos superado. El número máximo de sensores que se pueden cargar desde un archivo es 100.",
+        "MISSING_COLUMNS": "Se requieren columnas/faltan.",
+        "EXTERNAL_ID": "Identificación externa no válida, debe tener entre 1 y 20 caracteres.",
+        "SENSOR_NAME": "Nombre del sensor no válido, debe estar entre 1 y 100 caracteres.",
+        "SENSOR_LATITUDE": "Valor de latitud no válido, debe estar entre -90 y 90. y menos de 10 decimales.",
+        "SENSOR_LONGITUDE": "Valor de longitud no válido, debe estar entre -180 y 180. y menos de 10 decimales.",
+        "SENSOR_READING_TYPES": "Tipo de lectura no válido detectado, los valores válidos incluyen: soil_water_content, soil_water_potential, temperature.",
```

Comment:
Could you check this translation? {{ reading_types }} needs to be included.
English: `"Invalid reading type detected: {{ reading_types }}. Valid values include: soil_water_content, soil_water_potential, temperature."`